src/v/kafka/server/group.cc

```diff
@@ -2786,15 +2786,15 @@ ss::sstring group_state_to_kafka_name(group_state gs) {
     // since these states are returned through the kafka describe groups api.
     switch (gs) {
     case group_state::empty:
-        return "Empty";
+        return ss::sstring(group_state_name_empty);
```

Comment:
converting to ss::sstring here isn't desired in general, but without more downstream changes this is required since ss::sstring doesn't have an implicit conversion from std::string_view.

Reply:
:sadface: